src/DTS/Repository/TemplateRepository.php

```diff
@@ -37,7 +37,7 @@ public function createNewTemplate(string $docType, string $key, string $author,
 
   public function createNewCompiledTemplate(Template $template, string $compiled): CompiledTemplate {
     return CompiledTemplate::fromState([
-        'templateId' => $template->templateId,
+        'templateId' => $template->id,
```

Comment:
id?

Reply:
This is for compiled templates. Since compiled templates have no ID on their own, they're using templateId. I like that name here as it shows which ID we're storing.